NSS-Day-Cohort-66/rare-api-golden-droids
# Zd rare first ticket

I added the method and serializer for UserPostView with Routes and Navbar complete


Supported route:
`/posts` for GET request that returns all posts for individual user based on token

_**Steps to Test**_

1. checkout to main and git fetch
2. switch to zd-rare-first-ticket branch
3. restart debugger if running
4. Open Postman
5. Get request `http://localhost:8000/posts` and verify you get an array of posts in response with 200 code.
6.verify following keys :

-       "id":
        "title": 
        "rare_user": 
        "category": 
        "publication_date": 
        "content":
